src/app/loadout-drawer/loadout-drawer-reducer.ts

```diff
@@ -194,64 +213,80 @@ export function removeItem(
   });
 }
 
+function setEquipForItemInLoadout(
```

Comment:
Would be great to have documentation on this

Reply:
Can do

Reply:
Done